tests/test_func/bad/bad_func.catala_en

```diff
@@ -33,17 +33,17 @@ $ catala Interpret -s S
 There is a conflict between multiple valid consequences for assigning the same variable.
 
 This consequence has a valid justification:
-┌─⯈ tests/test_func/bad/bad_func.catala_en:14.65-14.70:
+┌─⯈ tests/test_func/bad/bad_func.catala_en:14.67-14.68:
 └──┐
 14 │   definition f of x under condition (x >= x) consequence equals x + x
-   │                                                                 ‾‾‾‾‾
+   │                                                                   ‾
```

Comment:
So this is a little bit unfortunate. I have the impression that when you build the `EAppOp`, you take the mark of the operator rather than taking the mark of the entire expression. I would be in favor of reverting to the previous behavior.